ess/src/API/EMBC.ESS/Managers/Events/EventsManager.Supports.cs

```diff
@@ -409,4 +415,63 @@ public async Task<EligibilityCheckQueryResponse> Handle(EligibilityCheckQuery qu
             Eligibility = eligibility
         };
     }
+
+    public async Task<bool> SendEmailConfirmation(ETransferDetails eTransferDetails, string primaryRegistrantId, string primaryRegistrantUserId, IEnumerable<Shared.Contracts.Events.Support> supports)
```

Comment:
should be private. It will affect the tests which should be unit, not integration